eclass/latex-package.eclass

```diff
@@ -206,10 +189,10 @@ latex-package_src_doinstall() {
 # relevant files that will be installed
 latex-package_src_compile() {
 	debug-print function $FUNCNAME $*
-	for i in `find \`pwd\` -maxdepth 1 -type f -name "*.ins"`
+	find $(pwd) -maxdepth 1 -type f -name "*.ins" | while read i;
```

Comment:
Why do you need `pwd` here? You aren't changing the directory or anything, so I don't see why leaving the path out would do any difference.